pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -433,84 +455,105 @@ const StorageRow = ({ connectionName, storageSize, storageSizeUnit, onValueChang
 
     return (
         <>
-            <label className="control-label" htmlFor="storage-pool-select">
-                {_("Storage")}
-            </label>
-            <Select.Select id="storage-pool-select"
+            <Expandable id='storage-details'
+                    toggleText={expandedStorage ? 'Hide Storage Details' : 'Show Storage Details'}
+                    onToggle={() => onValueChanged('expandedStorage', !expandedStorage)}
+                    isExpanded={expandedStorage}
+                    className='ct-form-full'>
+                <span />
+            </Expandable>
+            {expandedStorage && <>
+                <label className="control-label" htmlFor="storage-pool-select">
+                    {_("Storage")}
```

Comment:
Same here -- how much disk you need can't be known by the OS vendor, libvirt, or Cockpit. Let's please show this by default.

Reply:
Let's continue the discssion on the memory comment, it's the same issue anyway.

Reply:
Agreed, resolving this one.